boards/st/nucleo_h723zg/nucleo_h723zg.dts

```diff
@@ -190,3 +191,36 @@ zephyr_udc0: &usbotg_hs {
 &rng {
 	status = "okay";
 };
+
+&flash0 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		/* 128KB for bootloader */
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x00000000 DT_SIZE_K(128)>;
+			read-only;
+		};
+
+		/* primary application image slot: 384KB */
+		slot0_partition: partition@20000 {
+			label = "image-0";
+			reg = <0x00020000 DT_SIZE_K(384)>;
+		};
+
+		/* secondary application image slot: 384KB */
+		slot1_partition: partition@80000 {
+			label = "image-1";
+			reg = <0x00080000 DT_SIZE_K(384)>;
+		};
+
+		/* swap slot: 128KB */
+		scratch_partition: partition@e0000 {
+			label = "image-scratch";
+			reg = <0x000e0000 DT_SIZE_K(128)>;
```

Comment:
Not blocking:
Scratch partition is no more required as a `BOOT_SWAP_USING_MOVE` option is available in MCUBoot which will be used by default if scratch partition is not present.
This has the benefit of providing more space to application and limit this sector wear out.

Reply:
Thanks for reviewing.  
If you think that would be more useful, I can replace the scratch partition with a storage partition of the same size.